Automattic/wp-calypso
# Seller Experience: Add 'Simple' vs 'More Power' view

#### Changes proposed in this Pull Request

* Depends on [PR 60222](https://github.com/Automattic/wp-calypso/pull/60222)

* Added new view so the user can pick from a simpler (Payment Blocks) or more powerful (Woo) store.

#### Testing instructions



* Access `start/setup-site/intent?siteSlug=<SITE SLUG>&flags=seller-experience` (which enables the `seller-experience` feature flag)
* Click the "Start Selling" button
![image](https://user-images.githubusercontent.com/3801502/150015569-2747a047-f39c-4373-b577-f9d4d95a1402.png)
* Click "Continue"

![image](https://user-images.githubusercontent.com/3801502/150198467-685f83bd-629e-4cb6-9b3f-8fd13dfa31bb.png)

* Check the new view
![image](https://user-images.githubusercontent.com/3801502/150831004-563f84a9-64c2-4c6c-b5ef-e3b62920a70d.png)



Related to #60155
